src/backend/aspen/cli/tests/test_db_cli.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path, PosixPath
+from tempfile import NamedTemporaryFile
+
+from click.testing import CliRunner
+
+from aspen.cli import db
+from aspen.database.connection import SqlAlchemyInterface
+from aspen.workflows.pangolin.tests.test_pangolin_workflow import create_test_data
+
+
+def test_create_mega_fasta(mocker, session, postgres_database):
+
+    create_test_data(session)
+    public_identifiers_txt: PosixPath = Path(
```

Comment:
```suggestion
    public_identifiers_txt = Path(
```

someone could run this on Windows. 😮

Reply:
lololol